File: src/main/java/wang/crick/ddl2plantuml/model/Table.java
package wang.crick.ddl2plantuml.model;

import lombok.Data;

import java.io.Serializable;
import java.util.List;

/**
 * 表
 *
 * @author wangyuheng
 * @date 2018/6/28 14:04
 */
@Data
public class Table implements Serializable {

    private String name;
    private String comment;

    private List<Column> columnList;
    private List<Index> indexList;
}

File: src/main/kotlin/wang/crick/ddl2plantuml/model/Table.kt
package wang.crick.ddl2plantuml.model

import java.io.Serializable

/**
 * @author wangyuheng
 * @date 2019-01-20 12:54
 */
class Table(var name: String, var comment: String, var columnList: List<Column>, var indexList: List<Index>) : Serializable